### DISCUSSION TITLE
The entity is no longer being provided by the tesla_custom integration

### DISCUSSION THREAD

**Discussion by Hubro** [alandtse/tesla#428] [Q&A] [answered]
It's been a while since last time, so I updated my tesla_custom integration to the latest version. (I ran `git pull` on the `dev` branch, deleted my existing tesla_custom directory and copied over the new one.)

I then restarted HAS.

![image](https://user-images.githubusercontent.com/597206/207299607-3da8ac7d-dc66-47e5-9e4b-40863dff36be.png)

![image](https://user-images.githubusercontent.com/597206/207299119-e2b05bb1-ef22-4272-9b86-22d21cfb5a1f.png)

The integration still seems to exist, still seems to be authenticated towards my Tesla account and I don't see any errors in the logs, however all 41 entities for my car has this same message:

> This entity is no longer being provided by the tesla_custom integration. If the entity is no longer in use, delete it in settings.

What did I do wrong? How do I fix this?

**Comment by alandtse** [comment, marked as answer]
Remove and readd. We had a major breaking release where entities were renamed.

**Reply by Hubro** [reply]
Yep, that did it
